en/includes/references/conditional-auth/api-reference.md

```diff
@@ -1332,15 +1340,77 @@ It contains the necessary metadata for invoking the API when calling the callCho
   </tr>
 </table>
 
-If the consumer key and the consumer secret are added as secrets, they should be included in the ConnectionMetadata as aliases, as shown below.
+You can securely store consumer keys and secrets as **secrets** in conditional authentication scripts and refer to
+them in your conditional authentication scripts using the `secrets.key` syntax. For example, to retrieve a secret value, you may use:
+```angular2html
+var consumerSecret = secrets.clientSecret;
+```
+For more information on adding secrets, refer to the [Add a secret to the script]({{base_path}}/guides/authentication/conditional-auth/configure-conditional-auth/#add-a-secret-to-the-script) section in the 
+documentation.
+
+??? note "Change in behavior from 30th June 2024"
+    Starting from 30th June 2024, you are no longer required to set the aliases for consumer keys and consumer secrets when calling the callChoreo command. Instead, you may directly reference them using the `secrets.secretName` notation.
+    However, if you prefer, you may continue using the previous method as follows.
+      <table>
+          <tr>
+            <td><code>connectionMetadata.consumerKeyAlias</code></td>
+            <td>The name of the secret that stores the consumer key.</td>
+          </tr>
+          <tr>
+            <td><code>connectionMetadata.consumerSecretAlias</code></td>
+            <td>The name of the secret that stores the consumer secret.</td>
+          </tr>
+      </table>
+
+### AuthConfig
+
+When using httpGet or httpPost functions in Asgardeo adaptive authentication scripts, the table summarizes each
+authentication type and its required properties:
+
+<table>
+    <thead>
+        <tr>
+            <th>Authentication Type</th>
+            <th>Properties</th>
+            <th>Description</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>basicauth</td>
+            <td>username, password</td>
+            <td>Uses user credentials.</td>
+        </tr>
+        <tr>
+            <td>apikey</td>
+            <td>apiKey, headerName</td>
+            <td>Uses an API key sent as a header.</td>
+        </tr>
+        <tr>
+            <td>clientcredential</td>
+            <td>consumerKey, consumerSecret, tokenEndpoint, scope (optional, a space separated list of scopes)</td>
+            <td>Uses client credentials to obtain an access token.</td>
+        </tr>
+        <tr>
+            <td>bearertoken</td>
+            <td>token</td>
+            <td>Uses a bearer token for authentication.</td>
+        </tr>
+    </tbody>
+</table>
+
+You can securely store sensitive values of properties like username, password, consumerKey, consumerSecret as secrets in conditional authentication scripts and refer to them in your conditional authentication scripts using the `secrets.key` syntax. For example, to retrieve a secret value, you can use:
```

Comment:
Be consistent on this syntax format. refer to my comment above.

Reply:
Fixed with https://github.com/wso2/docs-is/pull/4773/commits/9acc22f1d66873e2c67cb71939862187674bece9